blocks/results-list-block/features/results-list/results/mock-data.js

```diff
@@ -0,0 +1,296 @@
+export default {
+	_id: "55FCWHR6SRCQ3OIJJKWPWUGTBM",
+	type: "story",
+	version: "0.10.4",
+	content_elements: [
+		{
+			_id: "L6RMSXTS7RB5XFKZPPAZKXGRF4",
+			type: "text",
+			additional_properties: [Object],
```

Comment:
All these `[Object]`s will just cause issues if anything tried to access that. I'd rather we pare the test data down to only the data we need for the test condition(s).  (I'm assuming this was copy/pasted from somewhere?)